### PR TITLE
docs: add markdown for hssi, opaevfio, and opaeuio

### PR DESCRIPTION
**Pull request by tswhison**
Signed-off-by: Tim Whisonant <tim.whisonant@intel.com>

### CODE REVIEW

**Comment by r-rojo** [comment]
Were you planning on adding these to the TOC?

**Comment by tswhison** [comment]
Good call.  Make sure I did it correctly.

**Review by anandaravuri — APPROVED** [review]

**Review by r-rojo — DISMISSED** [review]

**Review by anandaravuri — APPROVED** [review]

File: doc/src/fpga_tools/opaevfio/opaevfio.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+`opaevfio [-h] [-i] [-r] [-d DRIVER] [-u USER] [-g GROUP] [-n] [-v] [addr]`
+
+## DESCRIPTION ##
+
+The ```opaevfio``` command enables the binding/unbinding of a PCIe device
+to/from the vfio-pci device driver. See https://kernel.org/doc/Documentation/vfio.txt
+for a description of vfio-pci.
+
+`addr`
```
Comment (r-rojo) [inline doc/src/fpga_tools/opaevfio/opaevfio.md:13]: Shouldn't we have an ##OPTIONS## sections?

Comment (tswhison) [reply doc/src/fpga_tools/opaevfio/opaevfio.md:13]: Is there precedent in some of the other .md files we've written.  I don't seem to see it.  Point me towards what you're thinking about.

**Review by r-rojo — APPROVED** [review]

**Review by anandaravuri — APPROVED** [review]

**Pull request merged**